week3project.py

```diff
@@ -0,0 +1,77 @@
+import zipfile2 as zipfile
+
+from PIL import Image
+import pytesseract
+import cv2 as cv
+import numpy as np
+
+face_cascade = cv.CascadeClassifier('haarcascade_frontalface_default.xml')
```

Comment:
If you want other people to be able to run your code, you need to add this .xml file to the repo